docs/tutorials/roblox/parenting.md

```diff
@@ -121,7 +121,7 @@ local folder = scope:New "Folder" {
             Color = Color3.new(1, 0, 0)
         },
         -- state object containing children (or nil)
-        scope:Computed(function(use)
+        scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
        scope:Computed(function(use)
```

Reply:
reverted